openhantek/src/configdialog/DsoConfigProbePage.h

```diff
@@ -0,0 +1,47 @@
+//
```

Comment:
I forgot to add this information to the contribution guideline, but please use the license header line